### PR TITLE
Add e2e test to verify that GPU pool is not scaled up if GPUs are not requested

### PR DESCRIPTION
**Pull request by losipiuk**
**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: test/e2e/autoscaling/cluster_size_autoscaling.go [hunk 1]
```diff
@@ -253,6 +253,32 @@ var _ = SIGDescribe("Cluster size autoscaling [Slow]", func() {
...
+		defer deleteNodePool(gpuPoolName)
+
+		installNvidiaDriversDaemonSet()
+
+		By("Enable autoscaler")
+		framework.ExpectNoError(enableAutoscaler(gpuPoolName, 0, 1))
+		defer disableAutoscaler(gpuPoolName, 0, 1)
+		Expect(len(getPoolNodes(f, gpuPoolName))).Should(Equal(0))
+
+		By("Schedule buch of pods beyond filling default pool but do not request any GPUs")
```
Comment (aleksandra-malinowska) [inline test/e2e/autoscaling/cluster_size_autoscaling.go]: bunch

Comment (losipiuk) [reply test/e2e/autoscaling/cluster_size_autoscaling.go]: doen

**Pull request merged**